client/state/happychat/middleware-calypso.js

```diff
@@ -128,7 +127,7 @@ export const sendActionLogsAndEvents = ( { getState, dispatch }, action ) => {
 	}
 
 	// If there's analytics metadata attached to this action, send analytics events
-	if ( has( action, 'meta.analytics' ) ) {
+	if ( action.meta?.analytics ) {
```

Comment:
Seems like previously if we specified `analytics: false`, it would still record analytics. Are we sure this was not the intended behavior?

Reply:
In practice `action.meta.analytics` is always an array, when present. The analytics middleware relies on that, calling `.forEach` on it without guarding.